embassy-nrf/src/lib.rs

```diff
@@ -267,5 +268,12 @@ pub fn init(config: config::Config) -> Peripherals {
     #[cfg(feature = "_time-driver")]
     time_driver::init(config.time_interrupt_priority);
 
+    // Disable UARTE (enabled by default for some reason)
```

Comment:
Are you sure this is not an SPM bug? 

Either way, could you split this to a separate PR?

Reply:
Not sure, tried a couple different SPM:s with same result but they may all have the same bug... Will submit this as a separate PR (note that SPIS is unusable without this)